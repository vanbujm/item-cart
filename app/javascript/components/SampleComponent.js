import React, {Component} from 'react';

class SampleComponent extends Component {
    render() {
        return (
            <h1>Hi, I'm SampleComponent! HOT RELOAD IS THE BEST</h1>
        );
    }
}

export default SampleComponent;