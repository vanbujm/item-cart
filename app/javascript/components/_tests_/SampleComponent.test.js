import React from 'react';
import SampleComponent from '../SampleComponent';
import {shallow} from 'enzyme';

test('SampleComponent renders', () => {
    const component = shallow(
        <SampleComponent/>
    );
    expect(component.html()).toMatchSnapshot();
});