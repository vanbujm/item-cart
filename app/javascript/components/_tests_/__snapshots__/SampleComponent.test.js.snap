// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SampleComponent renders 1`] = `
<h1>
  Hi, I'm SampleComponent!
</h1>
`;
